### PR TITLE
split benign paroxysmal positional nystagmus

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
address #962

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@cmungall -
this OMIM record had the generic type of the disease and type 1 class on the same record, but there is not a OMIMPS that groups the type 1 and type 2.

I split the generic and type 1 and kept the OMIM:equivalentTo on the generic term and added the OMIM xref to the the type 1 term (which is a sibling) and added the annotation MONDO:directSiblingOf.

Is this the correct approach?

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -438204,6 +438184,44 @@ is_a: MONDO:0100249 {source="OMIM:400045"} ! 46,XX testicular disorder of sex de
...
+synonym: "vertigo, benign paroxysmal positional" RELATED [OMIM:193007]
+synonym: "vertigo, benign recurrent" RELATED [MONDO:Lexical, OMIM:193007]
+synonym: "vertigo, benign recurrent; BRV" RELATED [OMIM:193007]
+synonym: "vestibulopathy, familial" RELATED [OMIM:193007]
+xref: COHD:81878 {source="MONDO:equivalentTo"}
+xref: DOID:13941 {source="MONDO:equivalentTo"}
+xref: GARD:0005915 {source="MONDO:equivalentTo", source="OMIM-shared"}
+xref: ICD9:386.11 {source="MONDO:equivalentTo", source="i2s", source="DOID:13941"}
+xref: MESH:D065635 {source="DOID:13941"}
+xref: OMIM:193007 {source="MONDO:equivalentTo", source="DOID:13941"}
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:438214]: the OMIM entry does not mention nystagmus

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:438214]: The MESH (MESH:D065635) definition is:
Idiopathic recurrent VERTIGO associated with POSITIONAL NYSTAGMUS. It is associated with a vestibular loss without other neurological or auditory signs. Unlike in LABYRINTHITIS and VESTIBULAR NEURONITIS, inflammation in the ear is not observed.
(and this is what was used on the Mondo record).

I think this label came from DO (https://www.ebi.ac.uk/ols/ontologies/doid/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FDOID_13941).

Should we:
1) have a term labeled 'benign paroxysmal positional nystagmus' with this def that xrefs MESH (and UMLS:C0155502, as they use the same def)
2) have a separate term labeled VERTIGO, BENIGN RECURRENT that xrefs [OMIM](https://omim.org/entry/193007) and [GARD](https://rarediseases.info.nih.gov/diseases/5915/benign-paroxysmal-positional-vertigo)?
3) should VERTIGO, BENIGN RECURRENT, 1 be a it's own Mondo class or should it be a synonym on the Mondo term that is equiv to OMIM:193007 VERTIGO, BENIGN RECURRENT (note OMIM has both of these terms on the same record).

**Comment by nicolevasilevsky** [comment]
Note to self, these records are xref'd from this class:
OMIM: https://omim.org/entry/193007
MESH: https://meshb.nlm.nih.gov/record/ui?ui=D065635
GARD: https://rarediseases.info.nih.gov/diseases/5915/benign-paroxysmal-positional-vertigo
DO: https://www.ebi.ac.uk/ols/ontologies/doid/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FDOID_13941
SNOMED: https://browser.ihtsdotools.org/?perspective=full&conceptId1=404684003&edition=MAIN/2021-01-31&release=&languages=en
UMLS: https://uts.nlm.nih.gov/uts/umls/concept/C0155502

**Comment by nicolevasilevsky** [comment]
Moved these questions to a ticket. Closing this PR

**Pull request closed without merging**